### ISSUE TITLE
Change the default branch

### ISSUE THREAD

**Issue by vkeranov** [dompdf/dompdf#978]
This is not an issue, it's something like a proposal. The latest commit in master branch is more than one year old. Some people just visit the github page and they think the project is not active. As far as I know the "master" branch is for version 0.6.x. The active development now is for 0.7.0, so why not just changing the default branch to "develop"?

**Comment by bsweeney** [comment]
I was hoping to have 0.7.0 out by now, but some requested changes that make total sense are still in the pipeline. My current thought is to finish with the current todo list, release another beta, then if no major issues crop up merge back into master. Final release would be shortly after that.

In the future I'd like to see us do major development (e.g. improved support for tables, svg, and float) on their own branches (to be eventually merged into develop) and use the develop branch for minor fixes and tweaks so that we can put out minor releases more often.

**Comment by bfrohs** [comment]
It seems strange doing the development on the non-master branch after the merge of 0.7.0. I understand feature branches, but merging those into a separate branch when they're merged hides them from the general public (I almost thought the project was abandoned until I looked through the branches).

I'm guessing you're following the branching model described in http://nvie.com/posts/a-successful-git-branching-model/?

You should be able to follow the same model, with only a slight modification to make the project look more active on GitHub: rename the `master` branch to `release-0.6.1` and rename `develop` to `master`. Or simply set `develop` as the main branch. I think it's pretty well-established that the `master` branch has new code that may or may not be production-ready, and production code should use a tagged release.

**Comment by bsweeney** [comment]
I was looking at the post on nvie.com as a model. The main reason we went down this path was due to requests from the community. Certainly I understand the desire to develop off the master branch. It is a bit frustrating that github doesn't highlight activity on other branches in the activity graph. However, unless there's a strong request to move back towards active development on the master branch I don't know that we'll switch back now.

At any rate I hope the question will be moot once 0.7 is out the door. The whole reason I want to start working off of feature branches is so that we can release bug fix versions more often. There are so many minor issues that get caught up because we're working on one major change or another.

Of course, all community feedback on this point is welcome.
